README.md

```diff
@@ -331,4 +336,8 @@
 
 ## Project Management
   
-  * [milestones](https://github.com/turbopape/milestones) 
+  * [milestones](https://github.com/turbopape/milestones)
+ 
+## Products
+
+  * [Metabase](http://www.metabase.com/)
```

Comment:
@RenanPalmeira please, change link to https://github.com/metabase/metabase and put this to the top under "Awesome products in Clojure"